### DISCUSSION TITLE
Getting started example "dep pwd" returns only "/"

### DISCUSSION THREAD

**Discussion by rowild** [deployphp/deployer#2307] [Help needed]
- Deployer version: 6.8
- Deployment OS: Mac os 10.13.6

Working through "Getting stared": at

```
dep pwd
```
the return value is always "/". 
But running `pwd` on the remote host does return a path.

Why could that be?

**Comment by antonmedv** [comment]
The / is also a path. Of root.

**Comment by rowild** [comment]
It is supposed to run on remote. There a different result is returned (complete path from root to folder). According to the example in the "Getting started" guide such a path  the path that is generated on remote - should be returned.

**Comment by antonmedv** [comment]
Try to run cd to desired dir before pwd.
